master/internal/resourcemanagers/kubernetes/pods.go

```diff
@@ -597,6 +602,28 @@ func (p *pods) summarize(ctx *actor.Context) map[string]model.AgentSummary {
 			}
 		}
 
+		for _, taskName := range nodeToTasks[node.Name] {
+			for i := int64(0); i < taskSlots[taskName]; i++ {
+				if curSlot >= int(numSlots) {
+					ctx.Log().Warnf("too many pods mapping to node %s", node.Name)
+					continue
+				}
+
+				slotsSummary[strconv.Itoa(curSlot)] = model.SlotSummary{
+					ID:      strconv.FormatInt(i, 10),
```

Comment:
I think `Itoa` is a more popular way to this in out codebase?
```suggestion
					ID:      strconv.Itoa(i),
```

Reply:
yeah `Itoa` doesn't work with int64? So I used `FormatInt` instead